src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -3,15 +3,20 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import argparse
+import sys
 
+import argparse
 import argcomplete
 
 import azure.cli.core.telemetry as telemetry
 import azure.cli.core._help as _help
 from azure.cli.core._util import CLIError
 from azure.cli.core._pkg_util import handle_module_not_installed
 
+import azure.cli.core._logging as _logging
```

Comment:
We should make the `_logging` module public. It is referenced everywhere. It doesn't make sense keep it a private module.